<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptChat</class>
 <widget class="QWidget" name="OptChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptChatUI</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="ck_chatSoftReturn">
     <property name="text">
      <string>Enter key sends chat messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_alertOpenChats">
     <property name="text">
      <string>Alert chat messages even if chat window is open</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_raiseChatWindow">
     <property name="text">
      <string>Raise chat window on receiving new messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_switchTabOnMessage">
     <property name="text">
      <string>Switch tab on receiving new messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_smallChats">
     <property name="text">
      <string>Open chats in compact mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox18">
     <property name="title">
      <string>Default action</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_defActMsg">
        <property name="text">
         <string>&amp;Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_defActChat">
        <property name="text">
         <string>&amp;Chat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox17">
     <property name="title">
      <string>Delete chat window contents</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_delChatsClose">
        <property name="text">
         <string>&amp;When chat window is closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsHour">
        <property name="text">
         <string>&amp;One hour after chat window was closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsDay">
        <property name="text">
         <string>One &amp;day after chat window was closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsNever">
        <property name="text">
         <string>&amp;Never</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
